.cookiecutter/includes/tox/passenv

```diff
@@ -10,7 +10,6 @@ dev: USE_HTTPS
 dev: NEW_RELIC_APP_NAME
 dev: NODE_ENV
 dev: PROXY_AUTH
-dev: ORCID_HOST
```

Comment:
This has moved into `setenv` (see below). (It was previously in `passenv` because it was set in devdata, but I've removed it from devdata because it's not a sensitive valie.)